st2common/setup.py

```diff
@@ -50,5 +50,6 @@
         'bin/st2-run-pack-tests',
         'bin/st2ctl',
         'bin/st2-generate-symmetric-crypto-key',
+        'bin/migrations/',
```

Comment:
might need some sort of glob pattern? see https://docs.python.org/2/distutils/sourcedist.html#manifest